lib/worker-transformer.js

```diff
@@ -28,10 +28,18 @@ class WorkerCallbackTransformer extends Transformer {
 
     compute(input, output) {
         if(typeof this._callback !== "function"){
-            throw new Error("renditionCallback must be a function");
+            throw new Error("compute: renditionCallback must be a function");
         }
         return this._callback(input, output, input.params);
     }
+
+    computeAllAtOnce(renditionsCallback) {
```

Comment:
this function needs to be called `compute` since that is what the transformer api expects.

I would create a new worker-transformer called `batchWorkerTransformer` that has a `compute` function that passes the correct params to the batchWorker cb: https://github.com/adobe/asset-compute-sdk#rendition-callback-for-batchworker-required
something like this:
```js
compute(input, output) {
    const outputs = [output]; // batch worker expects a list of renditions
    const outdir =  output.directory; // batch worker expects an output directory: https://git.corp.adobe.com/nui/asset-compute-pipeline/blob/mob-session/lib/sdk/rendition.js#L66
    return this._callback(input, outputs, outdir, params);
}
```